sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -563,7 +564,7 @@ for doc, action_results in zip(documents, document_results):
 
 The returned response is an object encapsulating multiple iterables, each representing results of individual analyses.
 
-Note: Multiple analysis is available only in the v3.1 API version.
+Note: Multiple analysis is available in API version v3.1 and up.
```

Comment:
Just curious: are we not planning to have an extraction summarization example in the readme? makes sense if we don't, since you said it's not the most popular scenario

Reply:
I remember Johan wanting to cut down the number of actions we show in the readme example. @maririos @deyaaeldeen @mssfang do you plan to show it?

Reply:
I am not planning to show it in readme or in the samples but I am open to do so if there is a consensus.

Reply:
interesting. I got similar feedback in .NET to remove the number of actions in the sample (we haven't done it :P ).
I will say, we should show 2, a common one, and consider one of the new ones as this is the only way we will showcase them.
Because I currently only know about Extractive Summarization I will suggest that one... but idk after there are other ones.
Another thing we can do is to leave as is for now, and ask the service team to tell us which are the actions that are more commonly used (based on data) and then show that.

Reply:
I only have an sycn/async Extractive Summarizaiton example in the sample folder but not show them in the README. I only mentioned it in terms.

Reply:
^ same here. Going to merge this and decide on sample strategy later.